packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.tsx

```diff
@@ -45,7 +45,7 @@ export const RadioGroup = compose<RadioGroupType>({
         // @ts-ignore - TODO, fix typing error
         contextValue.buttonKeys = React.Children.map(children, (child: React.ReactChild) => {
           if (React.isValidElement(child)) {
-            return child.props.buttonKey;
+            return child.props.value;
```

Comment:
I think this means that buttonKey is not used as a prop anymore on the Radio.

Reply:
We renamed buttonKey prop to value in V1 Radio